### ISSUE TITLE
Accessibility section

### ISSUE THREAD

**Issue by sifferhans** [sifferhans/basic-ui#28]
**Describe the solution you'd like**
There should be a section in the docs for a11y advice and how-tos. Should highlight some low-hanging fruit to implement.
